src/Microsoft.CodeAnalysis.AnalyzerUtilities/PublicAPI.Unshipped.txt

```diff
@@ -709,3 +709,12 @@ virtual Microsoft.CodeAnalysis.FlowAnalysis.DataFlow.DataFlowOperationVisitor<TA
 virtual Microsoft.CodeAnalysis.FlowAnalysis.DataFlow.DataFlowOperationVisitor<TAnalysisData, TAnalysisContext, TAnalysisResult, TAbstractAnalysisValue>.VisitUnaryOperatorCore(Microsoft.CodeAnalysis.Operations.IUnaryOperation operation, object argument) -> TAbstractAnalysisValue
 virtual Microsoft.CodeAnalysis.FlowAnalysis.DataFlow.PredicatedAnalysisData<TKey, TValue>.ApplyPredicatedData(Microsoft.CodeAnalysis.FlowAnalysis.DataFlow.DictionaryAnalysisData<TKey, TValue> coreAnalysisData, Microsoft.CodeAnalysis.FlowAnalysis.DataFlow.DictionaryAnalysisData<TKey, TValue> predicatedData) -> void
 virtual Microsoft.CodeAnalysis.FlowAnalysis.DataFlow.PredicatedAnalysisData<TKey, TValue>.RemoveEntryInPredicatedData(TKey key, Microsoft.CodeAnalysis.FlowAnalysis.DataFlow.DictionaryAnalysisData<TKey, TValue> predicatedData) -> void
+Analyzer.Utilities.Unit
```

Comment:
Please make this an internal type and revert the changes to the PublicAPI files. We only want to expose public APIs for functionality, not for utilities from this package.

Reply:
Makes sense, I didn't pay attention here...